packages/cli/src/commands/build.ts

```diff
@@ -55,3 +42,34 @@ async function copyResource(from: string, to: string) {
     console.error(`${chalk.red('error')} - ${err}`)
   }
 }
+
+async function cleanup() {
+  // Remove `node_modules` from temporary directory after build
+  removeSync(`${tmpDir}/node_modules`)
+}
+
+async function normalizeBuildOutputDir() {
+  // Fix Next.js standalone build output directory
+  if (existsSync(`${tmpDir}/.next/standalone/.faststore`)) {
+    const standaloneBuildFiles = readdirSync(
+      `${tmpDir}/.next/standalone/.faststore`
+    )
+
+    await Promise.all(
+      standaloneBuildFiles.map((file) =>
+        moveSync(
+          `${tmpDir}/.next/standalone/.faststore/${file}`,
+          `${tmpDir}/.next/standalone/${file}`,
+          { overwrite: true }
```

Comment:
This makes the magic, right?

Reply:
Yes!! We need that to move every file/directory from the build output wrong directory (`.next/standalone/.faststore`) to the right directory (`.next/standalone/`) then remove the wrong folder just for cleaning purposes